### DISCUSSION TITLE
--audio-channel-map  vs.  --audio-filter channel_remap

### DISCUSSION THREAD

**Discussion by alatteri** [CESNET/UltraGrid#401] [General]
What is the difference or advantages of `--audio-channel-map`  versus the new  `--audio-filter channel_remap module`?

**Comment by mpiatka** [comment]
The difference is when they are applied: ``--audio-channel-map`` works on the receiver before playback, whereas the ``--audio-filter channel_remap`` works on the sender, like the other audio filters, before compressing and sending.

**Reply by alatteri** [reply]
Thanks Martin.
